### DISCUSSION TITLE
Has RSSHub given up on Facebook?

### DISCUSSION THREAD

**Discussion by stanley0010** [DIYgod/RSSHub#16090] [Q&A]
I noticed there is no Facebook route on https://docs.rsshub.app/routes/ now. But I remember there was. Feel like anything about Facebook is wiped clean... Is RSSHub not gonna support (or try to support) Facebook anymore in the future?

**Comment by QAQDE** [comment]
Maintaining RSS services for Facebook is a tough and time-consuming habit :(
